src/DeviceRenderer.js

```diff
@@ -67,25 +67,31 @@ module.exports = class DeviceRenderer {
         this.x = 0;
         this.y = 0;
 
-        document.addEventListener('click', (event) => {
-            if (!this.hasSomeParentTheClass(event.target, 'gm-overlay')
-                && !event.target.classList.contains('gm-icon-button')
-                && !event.target.classList.contains('gm-dont-close')) {
-                this.emit('close-overlays');
-            }
-        });
+        document.addEventListener('click', this.clickHandlerCloseOverlay.bind(this));
+    }
+
+    /**
+     * Handler for click event, responsible for closing overlay when clicking outside of it.
+     * @param {PointerEvent} event Event that triggered this handler
+     */
+    clickHandlerCloseOverlay(event) {
+        if (!this.hasSomeParentTheClass(event.target, 'gm-overlay')
```

Comment:
The comment doesn't seem to be related to the scope of this pull request, but I'm wondering if hasSomeParentTheClass couldn't be refactored to something like

```
function hasSomeParentTheClass(element, className) {
    return element.closest("." + className) !== null;
}
```
https://developer.mozilla.org/en-US/docs/Web/API/Element/closest

Reply:
you're probably right, I'll do this right away in another PR.